modules/ssh/agent.go

```diff
@@ -110,7 +110,7 @@ func SshAgentWithKeyPairs(t *testing.T, keyPairs []*KeyPair) *SshAgent {
 
 // Instantiates and returns an in-memory ssh agent with the given KeyPair(s) already added
 // You should stop the agent to cleanup files afterwards by calling `defer sshAgent.Stop()`
-func SshAgentWithKeyPairsE(t *testing.T, keyPairs []*KeyPair) (*SshAgent, error) {
+func SshAgentWithKeyPairsE(t TestingT, keyPairs []*KeyPair) (*SshAgent, error) {
 	t.Log("Generating SSH Agent with given KeyPair(s)")
```

Comment:
It looks like you removed `Log` and `Logf` from the interface, but not the calls to `t.Log`/`t.Logf`?